src/server-code/main-server.js

```diff
@@ -160,36 +159,10 @@ export default function runMainServer(serverFalcorModel) {
     );
   });
 
+
```

Comment:
Now I might just be getting picky, but since I've requested other changes I'll also just make you remove this newline for consistency